icacls_work.rb

```diff
@@ -0,0 +1,36 @@
+class Rights
+
+  attr_reader :path, :name
+  def initialize(path, name)
+    @path = path
+    @name = name
+  end
+
+  def get_res_string(name_file)
+    system("cd #{@path} && icacls #{@name} > #{name_file}.txt")
```

Comment:
You shouldn't store output in text file here. Problem is with system command. Try it with 3 different approaches:
- backticks ``
- exec if its even possible (need to understand what exec does)
- Open3 library

Reply:
Use backticks `` that return the string